en/database.texy

```diff
@@ -36,7 +36,7 @@ Nette\Database will create own proper internal driver based on your database ser
 
 The 4th optional parameter of the [connection constructor |api:Nette\Database\Connection] allows you to specify some options. These options are passed to the PDO constructor and the internal driver constructor, so they can contain a configuration for both the PDO instance and the driver instance.
 
-All connections are created as "lazy" by default. This means the connection is established when it's needed, not when you create a `Connection` instance. You can disable this behavior by passing `'lazy' => FALSE` configuration.
+If you want that connection with the database server is not created immediately after an instance of `Connection` is created but only when the database is used, you can do so by setting the configuration to `'lazy' => TRUE`.
```

Comment:
If you want the connection to be established immediately after creating the instance of `Connection`, set `'lazy' => TRUE`.